feature/orchestrator/src/main/java/com/simprints/feature/orchestrator/OrchestratorActivity.kt

```diff
@@ -11,13 +12,21 @@ import com.simprints.infra.uibase.navigation.handleResult
 import com.simprints.infra.uibase.viewbinding.viewBinding
 import dagger.hilt.android.AndroidEntryPoint
 
+private const val KEY_IS_GRAPH_INITIALIZED = "KEY_IS_GRAPH_INITIALIZED"
```

Comment:
I would prefer to keep constants in companion objects to keep class-per-file rules.